src/rust/rust-proto/src/graplinc/grapl/api/graph_query/v1beta1/messages.rs

```diff
@@ -18,6 +18,21 @@ use crate::{
     SerDeError,
 };
 
+/// Basically cribbed from std::collection::HashMap's implementation.
```

Comment:
why are we using this non-standard hashmap?

Reply:
Great question. It's what I received in a Colin code-dump and never really questioned it. FWIW it appears to be semi-standard (they use it in rustc)?